### PR TITLE
IGAPP 173: (Re-)Update non-breaking libraries

### PR DESCRIPTION
**Pull request by Znippet**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

@react libraries will be updated in a different branch
(https://issues.tuerantuer.org/browse/IGAPP-484)

### CODE REVIEW

File: api-client/package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "@babel/runtime": "7.13.7",
     "js-md5": "^0.7.3",
     "entities": "^2.0.0",
-    "moment": "2.25.3",
+    "moment": "2.29.1",
```
Comment (klinzo) [inline api-client/package.json:23]: :fearful: Hopefully this does not break anything :D

File: native/package.json [hunk 2]
```diff
@@ -56,9 +56,9 @@
     "events": "^3.2.0",
     "hashids": "^2.2.8",
     "htmlparser2": "^6.0.0",
-    "i18next": "^19.1.0",
+    "i18next": "19.9.1",
```
Comment (klinzo) [inline native/package.json:59]: Why did you remove `^`?

Comment (klinzo) [reply native/package.json:59]: Same for most of the other dependencies

**Review by klinzo — APPROVED** [review]

File: native/package.json (hunk 2 above)
Comment (klinzo) [reply native/package.json:59]: I just recognized you did the same for babel dependencies in a previous PR, right?

File: api-client/package.json (hunk 1 above)
Comment (Taggotty) [reply api-client/package.json:23]: So I checked the release notes and it seems, that there were no breaking changes. However I found [this](https://momentjs.com/docs/#/-project-status/) interesting article about moment becoming a legacy project o.o

**Comment by klinzo** [comment]
Tested on firefox and real android 10 device.

**Review by Taggotty — APPROVED** [review]

File: api-client/package.json (hunk 1 above)
Comment (klinzo) [reply api-client/package.json:23]: Yeah crazy, right? We already have an issue to switch away from moment but we wanted to wait until we get correct timestamps from the cms which will only be the case with django.

**Comment by Taggotty** [comment]
roughly tested on android simulator.

File: native/package.json (hunk 2 above)
Comment (Znippet) [reply native/package.json:59]: I thing it doesn't make much of a difference. With the ^ yarn upgrade will only do minor version upgrades.
But with the lockfile it won't do anything automatically anyway. So if we upgrade we should try to upgrade to the latest version because testing is necessary either way.

The standard behavior of installing a new packet via yarn upgrade is adding the "^" by default. 
I can also re-add it, I don't care
